Documentation update with xdp_test (#3752)
microsoft/ebpf-for-windows: docs/InstallEbpf.md

    ```cmd
    kubectl apply -f manifests/Kubernetes/ebpf-for-windows-daemonset.yaml
    ```

## Installing eBPF external extensions

> - Install [xdp-for-windows](https://github.com/microsoft/xdp-for-windows/releases) extension, if your eBPF program uses XDP driver. Please refer to  [installation here](https://github.com/microsoft/xdp-for-windows/blob/main/docs/usage.md)

Reviewer: ```suggestion
- If your eBPF program requires XDP, install the [xdp-for-windows](https://github.com/microsoft/xdp-for-windows/releases) extension by following the [XDP for Windows installation instructions](https://github.com/microsoft/xdp-for-windows/blob/main/docs/usage.md).
```
grammar and punctuation.
